UI/setup/new_user.html

```diff
@@ -49,7 +47,7 @@
          label_pos = 1 ;
          ?>
     </div>
-    <div>
+    <div style="float:left">
```

Comment:
What problem does this change solve? Is it part of the problem that this PR solves? In what way?

Reply:
Whenever we discover that the user already exists, we void everything and ask to toggle import and enter all over again. That specific line points the message to the import selection to make it obvious.

Reply:
The PR as a whole addresses your point

> Whenever we discover that the user already exists, we void everything and ask to toggle import and enter all over again. 

Agreed with that. I'm not sure what that has to do with explicitly this `style="float:left"` change you added here. I don't understand what you mean by

> That specific line points the message to the import selection to make it obvious.

Is there a visual effect that it achieves which is related to the PR?

Reply:
Float the message right to the `Import user` selection.